gradle/java.gradle

```diff
@@ -12,6 +12,7 @@ compileJava {
 
 task testJar(type: Jar, dependsOn: testClasses, group: "build") {
     appendix = 'test'
+    classifier = 'tests'
```

Comment:
Potential danger: a few pieces of infrastructure (test shell scripts) could depend on exact name of artifact :/